### PR TITLE
OSDOCS-7050: cluster wide proxy doc issues

### PR DESCRIPTION
**Pull request by smunje1**
[OSDOCS-7050](https://issues.redhat.com/browse/OSDOCS-7050): cluster wide proxy doc issues

Aligned team: Service Delivery
OCP version for cherry-picking: enterprise-4.13+
Preview pages: [Click to see the build preview in your browser](https://63094--docspreview.netlify.app/openshift-rosa/latest/networking/configuring-cluster-wide-proxy#cluster-wide-proxy-prereqs_configuring-a-cluster-wide-proxy)
SME review **completed**: @paulczar 
QE review **completed**: @yingzhanredhat 
Peer review **completed**: @nalhadef

### CODE REVIEW

**Comment by smunje1** [comment]
Hello @paulczar
Could you please review and approve the PR if there are no comments? Thank you!

**Review by paulczar — APPROVED** [review]
looks great!

**Comment by smunje1** [comment]
Hello @xueli181114 / @yuwang-RH
Can I please get a QE review for this PR? Thank you!

**Comment by smunje1** [comment]
We have the QE approval in Jira's [comment](https://issues.redhat.com/browse/OSDOCS-7050?focusedId=22742080&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22742080).

**Pull request merged**
